lib/routes/acgvinyl/news.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/news/:limit?',
+    categories: ['anime'],
+    example: '/news/20',
+    parameters: { limit: 'the number of posts for a request' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.acgvinyl.com'],
+            target: '/news',
+        },
+    ],
+    name: 'News',
+    maintainers: ['williamgateszhao'],
+    handler,
+    url: 'http://www.acgvinyl.com/col.jsp?id=103',
+    zh: {
+        name: '黑胶新闻',
+    },
+};
+
+async function handler(ctx) {
+    const rootUrl = 'http://www.acgvinyl.com';
+
+    const list = await cache.tryGet(rootUrl, async () => {
+        const newsIndexResponse = await ofetch(`${rootUrl}/col.jsp?id=103`);
+        const $ = load(newsIndexResponse);
+        const newsIndexJsonText = $('script:contains("window.__INITIAL_STATE__")').text().replaceAll('window.__INITIAL_STATE__=', '');
+        const newsIndexJson = JSON.parse(newsIndexJsonText);
+
+        const newsListResponse = await ofetch(`${rootUrl}/rajax/news_h.jsp?cmd=getWafNotCk_getList`, {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/x-www-form-urlencoded',
+            },
+            body: [
+                `page=1`,
+                `pageSize=${ctx.req.param('limit') ?? 20}`,
```

Comment:
Do use adjust the limit in path parameter. Please use the [common search parameter](https://docs.rsshub.app/guide/parameters) instead.